dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -109,6 +109,7 @@
             <Color x:Key="SolidBackgroundFillColorTertiary">#282828</Color>
             <Color x:Key="SolidBackgroundFillColorQuarternary">#2C2C2C</Color>
 
+            <Color x:Key="SystemFillColorAttention">#50C0FF</Color> <!-- This color might be incorrect -->
```

Comment:
This was updated in a recent PR that was checked in. SystemFillColorAttention color resources were removed.

Reply:
removed